documentatie/use-cases/GSB-gebruikersdoelen.md

```diff
@@ -107,33 +107,29 @@ __uitbreidingen__:
 - OSV richt de server ook gelijk als client in. Willen wij dit ook?
 - Afzenderverificatie ontbreekt nog, want nog geen beslissing over oplossing.
 
-## GSB richt de applicatie in
+
+## De coördinator maakt de stembureaus aan.
```

Comment:
```suggestion
## De coördinator voegt stembureaus toe.
```

Reply:
Moeten we hier nog expliciet maken dat het toevoegen van stembureaus _per verkiezing_ moet gebeuren en niet op applicatieniveau is? Als er tegelijkertijd meerdere verkiezingen zijn (bv provinciaal + waterschap), dan moeten voor beide stembureaus worden toegevoegd (en de set kan per verkiezing verschillen)

Reply:
> Moeten we hier nog expliciet maken dat het toevoegen van stembureaus per verkiezing moet gebeuren en niet op applicatieniveau is?

Goed punt! En dat doet me denken aan: Wat als je per ongeluk de stembureaus importeert voor de verkeerde verkiezing? Ga je dan een paar honderd keer op "verwijder stembureau" klikken?

Reply:
> @jorisleker Ben het niet eens met de wijziging "aanmaken" -> "toevoegen". Deze use case gaat over dat je Abacus met nul stembureaus hebt en ze dan aanmaakt. Eventuele latere aanpassingen vallen niet binnen deze use case. "voegt stembureaus toe" suggereert voor mij dat deze use case ook kan gebeuren nadat de initiële set stembureaus is aangemaakt, maar dat je er later nog een paar toevoegt.
> 
> 
> 
> Wat vind je van "De coördinator voert de stembureaus in."?

Grappig. Dit is denk ik vooral semantisch. Voor mij klinkt "aanmaken" heel erg als vanaf scratch de stembureaus verzinnen. Vanuit het perspectief van de applicatie is dat ook zo, maar in het proces van de verkiezing zijn die stembureaus allang verzonnen en zelfs al aangemaakt in andere systemen. Dat bedoelde ik ook met toevoegen: je haalt ze uit de andere applicatie en voegt ze toe in abacus. "Invoeren" voelt voor mij dan weer net iets teveel als "ik ga met het toetsenbord aan de slag" en mist de "importeren" is ook een optie betekenis. Wat denk je van ".. zet de stembureaus in de applicatie"

Reply:
perfect! aangepast